tests/models/config.spec.ts

```diff
@@ -0,0 +1,449 @@
+import {Config} from '../../addon/ng2/models/config';
+
+const fs      = require('fs');
+const path    = require('path');
+const expect  = require('chai').expect;
+const schema  = path.resolve(process.cwd(), 'test.json');
```

Comment:
Just a path to the file that is used for testing purposes, it is created `beforeEach`.